packages/vapor/scss/components/navigation.scss

```diff
@@ -125,10 +125,20 @@
                             justify-content: space-between;
                             height: $navigation-menu-section-height;
                             margin-right: $navigation-right-margin;
+                            padding-right: 2px;
                             padding-left: $navigation-horizontal-space;
                             color: var(--navy-blue-80);
                             font-size: var(--default-font-size);
                             letter-spacing: 0.05em;
+                            border: 2px solid transparent;
```

Comment:
Is this one really necessary?
```suggestion
```

Reply:
yes, otherwise when the focus is triggered it shifts the content to the right because its adding a 2px border when there was no border before